JSDOMParser.js

```diff
@@ -308,6 +308,7 @@
       }
     }
     getElems(this);
+    elems._noLiveNodeListsPlease = true;
```

Comment:
Can you rename this to `isLiveNodeList`? Booleans that start with `no` are confusing when negated, and really this boolean doesn't describe the policy (like it's named) but it describes the behavior of this list.